crank/models.py

```diff
@@ -1,11 +1,59 @@
 from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
 
 # Create your models here.
 class Greeting(models.Model):
     when = models.DateTimeField('date created', auto_now_add=True)
 
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    email_confirmed = models.BooleanField(default=False)
+
+#begin Tony-Frank pair insert
+class Course_Listing_Table(models.Model):
+    courseid = models.IntegerField(primary_key=True)
+    coursenumber=models.CharField(max_length=10)
+    coursetitle=models.CharField(max_length=128)
+
+class Faculty_Table(models.Model):
+    facultyid = models.IntegerField(primary_key=True)
+    facultyname = models.CharField(max_length=128)
+
+class Course_Faculty_Table(models.Model):
+    course = models.ForeignKey(Course_Listing_Table)
+    faculty = models.ForeignKey(Faculty_Table)
+    course_faculty_pair = models.IntegerField(primary_key=True)
+
+class Student_Course_Rating_Table(models.Model):
+    uni = models.ForeignKey(User, null=True)
+    class_id = models.ForeignKey(Course_Faculty_Table)
+    student_course = models.IntegerField(primary_key=True)
+
+class Course_Rating_Table(models.Model):
```

Comment:
What’s the primary key for this table? Why not merge this and the student_course_rating_table?